calico-cloud/users/user-management.mdx

```diff
@@ -62,6 +62,52 @@ The Admin role provides broad administrative access for day-to-day configuration
 | Container Threat Detection      | view, edit         |
 | Dashboards                      | view, edit         |
 
+### User Admin
+
+The User Admin role has the ability to manage team members and their assigned roles.
+
+| Feature                         | Permission Level   |
+| :------------------------------ | :----------------- |
+| Service Graph and Flow Visualizer | -                |
+| Policies                        | -                  |
+| Nodes and Endpoints             | -                  |
+| Network Sets                    | -                  |
+| Managed Clusters                | -                  |
+| Compliance Reports              | -                  |
+| Timeline                        | -                  |
+| Alerts                          | -                  |
+| Kibana                          | -                  |
+| Image Assurance                 | -                  |
+| Manage Team                     | view, edit, delete |
```

Comment:
I notice that no other role has 'delete' for this feature. Should we also have 'delete' here in the Owner and Admin sections?

Reply:
I suppose in theory "edit" includes deletions, so for consistency I'll simply remove it here.